apps/builder/app/builder/features/style-panel/sections/box-shadows/utils.ts

```diff
@@ -0,0 +1,85 @@
+import type { LayersValue, StyleProperty } from "@webstudio-is/css-data";
```

Comment:
utils is bad name.
Here box-shadow-layers.ts

Also having logic tests would be good to add

Reply:
renamed, tests are coming